arm-servermanagement/2015-07-01-preview/swagger/servermanagement.json

```diff
@@ -2105,7 +2105,7 @@
       }
     },
     "PowerShellTabCompletionResults": {
-      "description": "an array of strings representing the different values that can be tabbed thru",
+      "description": "An array of strings representing the different values that can be tabbed through.",
```

Comment:
Given that we don't know the client, would "selected" be better than "tabbed through"?